bin/run_tests_for_modules.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "${BASH_SOURCE%/*}/.."
+
+for module in $(echo "$@" | tr , ' '); do
```

Comment:
if there can be multiple arguments, no need for split (`tr`)
if there can be only one argument (because of how .travis.yml works), no need for `"$@"`

```bash
test $# -eq 1
.... echo "$1" | tr ...
```